daprdocs/content/en/developing-applications/building-blocks/actors/actors-timers-reminders.md

```diff
@@ -146,6 +146,28 @@ If an invocation of the method fails, the timer is not removed. Timers are only
 - The executions run out
 - You delete it explicitly
 
+## Reminder data serialization format
+
+Actor reminder data is serialized to JSON by default. Dapr supports a protobuf seralization format for reminders data which, depending on throughput and size of the payload, can result in significant performance improvements, giving developers a higher throughput and lower latency. Another benefit is storing smaller data in the actor underlying database, which can result in cost optimizations when using some cloud databases.
```

Comment:
```suggestion
Actor reminder data is serialized to JSON by default. In Dapr v1.13 onwards supports a protobuf serialization format for reminders data which, depending on throughput and size of the payload, can result in significant performance improvements, giving developers a higher throughput and lower latency. Another benefit is storing smaller data in the actor underlying database, which can result in cost optimizations when using some cloud databases. A restriction with using protobuf serialization is that the reminder data can no longer be queried. 
```

Reply:
Can we indicate a level of perf guidance improvement. For example expect between 10-20% perf improvement, so this is more explicit in the guidance?

Reply:
No, we can't indicate a level of performance improvement. Its dependent on an app's relative payload size and throughput